.github/CODEOWNERS

```diff
@@ -105,4 +105,61 @@ TeachingBubble/   @micahgodbolt
 Toggle/  @phkuo
 Tooltip/  @micahgodbolt
 
+## Docs
```

Comment:
I think it makes sense to use the same code owners as the rest of the component for now, but over time we'll want to find design owners who can review the usage guidelines.